EnlightenMAUI.csproj

```diff
@@ -77,6 +77,16 @@
 	</ItemGroup>
 
 	<ItemGroup>
+	  <None Remove="libraries\SiG-785\2mmHDPE.csv" />
```

Comment:
A reminder that EnlightenMAUI is open-source, and we should be careful about rolling matching libraries into the distribution. These particular spectra are non-problematic, but that won't always be the case.

Reply:
this is a good note, I will make names opaque when needed (or maybe even completely anonymized) and probably tend towards not storing spectra in the repo